script/deploy/yield-vault/aave-v3/DeployAaveV3Factory.s.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import { AaveV3Optimism } from "aave-address-book/AaveV3Optimism.sol";
+import { AaveV3ERC4626Factory, IPool, IRewardsController } from "yield-daddy/aave-v3/AaveV3ERC4626Factory.sol";
+
+import { ScriptHelpers } from "../../../helpers/ScriptHelpers.sol";
+
+contract DeployAaveV3Factory is ScriptHelpers {
+  function run() public {
+    vm.startBroadcast();
+
+    if (block.chainid == OPTIMISM_CHAIN_ID) {
+      new AaveV3ERC4626Factory(
+        IPool(address(AaveV3Optimism.POOL)),
+        EXECUTIVE_TEAM_OPTIMISM_ADDRESS, // Reward recipient,
```

Comment:
Not sure if this should be the executive team...because we can't change it.

I think we should deploy a remote owner for governance on Optimism

Reply:
This address would only be used to claim rewards distributed by Aave.
Seems a bit overkill to deploy a remote owner.